gui.py

```diff
@@ -74,9 +74,22 @@ def wrap_in_section(content, section_classes="", container_classes="", div_class
         ddsih.DangerouslySetInnerHTML(
             f"""
 <h1 class="title is-3">Statewide Temperature Index</h1>
-<p class="camera-icon">Click the <span>
+<p>“Has it been cold lately in Alaska?” Answer: “It’s complicated.” Why? Alaska is a very large region with complex 
```

Comment:
Switch the " glyphs to HTML entities (`&ldquo; and &rdquo;`), always strive to be a stickler for [proper typography](https://practicaltypography.com/straight-and-curly-quotes.html).  (Carolyn and I are huge dorks about this).